cypress/e2e/hub/execution-environments.cy.ts

```diff
@@ -1,14 +1,60 @@
 import { hubAPI } from '../../support/formatApiPathForHub';
 import { ExecutionEnvironments } from './constants';
+import { randomString } from '../../../framework/utils/random-string';
 
-describe.skip('Execution Environments', () => {
+describe('Execution Environments', () => {
+  let remoteRegistryName = '';
   before(() => {
     cy.hubLogin();
   });
 
-  it('it should render the execution environments page', () => {
+  beforeEach(() => {
+    remoteRegistryName = `remote_registry_${randomString(3, undefined, { isLowercase: true })}`;
+
+    cy.createRemoteRegistry(remoteRegistryName);
+  });
+
+  it('can render the execution environments page', () => {
+    cy.navigateTo('hub', ExecutionEnvironments.url);
+    cy.verifyPageTitle(ExecutionEnvironments.title);
+  });
+
+  it('can add and delete a new execution environment', () => {
+    const eeName = `execution_environment_${randomString(3, undefined, { isLowercase: true })}`;
+    const upstreamName = `upstream_name_${randomString(3, undefined, { isLowercase: true })}`;
     cy.navigateTo('hub', ExecutionEnvironments.url);
     cy.verifyPageTitle(ExecutionEnvironments.title);
+    cy.getByDataCy('add-execution-environment').click();
+    cy.getByDataCy('name').type(eeName);
+    cy.getByDataCy('upstream-name').type(upstreamName);
+    cy.contains('[data-ouia-component-id="menu-select"]', 'Select registry')
+      .click()
+      .then(() => {
+        cy.contains('button[type="button"]', `${remoteRegistryName}`).click();
```

Comment:
Fails on this line in my local environment, probably because I have a lot of registries in the system.